jsonnet/thanos-mixin/alerts/receive.libsonnet

```diff
@@ -4,10 +4,27 @@
       {
         name: 'thanos-receive.rules',
         rules: [
+          {
+            alert: 'ThanosReceiveHttpRequestErrorRateHigh',
+            annotations: {
+              message: 'Thanos Receive {{$labels.job}} is failing to handle {{ $value | humanize }}% of requests.',
+            },
+            expr: |||
+              (
+                sum(rate(http_requests_total{code=~"5..", %(thanosReceiveSelector)s, handler="receive"}[5m]))
+              /
+                sum(rate(http_requests_total{%(thanosReceiveSelector)s, handler="receive"}[5m]))
+              ) * 100 > 5
```

Comment:
Ultimately this should probably be measured on something that runs in front of the receive component. Otherwise, we'll only see errors that receive component itself sees, but not the Gateway Errors, when the receive component is down.

Reply:
We can keep it anyway. Might simply not be the best place for this.

Reply:
I totally agree this is not complete, for checking actual errors for receive, those need to be checked on a node balancer or similar.